v_4_9_0/master_template.go

```diff
@@ -322,8 +297,7 @@ systemd:
       --kubeconfig=/etc/kubernetes/kubeconfig/kubelet.yaml \
       --node-labels="node.kubernetes.io/master,node-role.kubernetes.io/master,kubernetes.io/role=master,role=master,ip=${DEFAULT_IPV4},{{.Cluster.Kubernetes.Kubelet.Labels}}" \
       --v=2"
-      ExecStop=-/usr/bin/docker stop -t 10 $NAME
-      ExecStopPost=-/usr/bin/docker rm -f $NAME
+      ExecStop=-/usr/bin/pkill kubelet
```

Comment:
Why do we need this? Shouldn't systemd stop the kubelet?

Reply:
I'm not sure why that's there. I removed it and tested and the process was stopped just fine. Fixed!